README.md

```diff
@@ -49,12 +52,12 @@ in a Debian/Ubuntu based distribution run:
 sudo apt install libgif-dev libjpeg-dev libopenexr-dev libpng-dev libwebp-dev
 ```
 
-We recommend using a recent Clang compiler (version 7 or newer), for that
-install clang and set `CC` and `CXX` variables. For example, with clang-7:
+We recommend using a recent Clang compiler (at least version 6), for that
```

Comment:
we actually recommend 7 or newer. Clang 6 "works" but it has too many SIMD-related bugs.

Reply:
I changed it back.